### PR TITLE
Add test folder and basic setup for find-recipes

### PR DESCRIPTION
**Pull request by xinyue296**
- Add test folder, and settings in gradle
- Basic setup for find-recipes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,14 +33,21 @@ Currently, we are working to implement the following features.
 
 - Minigames
   - [X] Rock-paper-scissors
+  - [ ] Trivia
```
Comment (christianjokhaji) [inline README.md]: Please mark as completed. I'll extend functionality later. :-)

File: src/main/java/ca/unknown/bot/use_cases/Parser.java [hunk 2]
```diff
@@ -14,7 +13,6 @@ public class Parser {
      * @return a JsonObject for further data manipulation and/or access.
      */
     public static JsonObject parse(String json) {
-        JsonElement jsonElement = JsonParser.parseString(json);
-        return jsonElement.getAsJsonObject();
+        return new Gson().fromJson(json, JsonObject.class);
```
Comment (christianjokhaji) [inline src/main/java/ca/unknown/bot/use_cases/Parser.java:16]: Much cleaner! Thanks.

File: README.md (hunk 1 above)
Comment (xinyue296) [reply README.md]: Okay! No problem!

File: src/main/java/ca/unknown/bot/use_cases/Parser.java (hunk 2 above)
Comment (xinyue296) [reply src/main/java/ca/unknown/bot/use_cases/Parser.java:16]: No worries!

**Review by christianjokhaji — CHANGES_REQUESTED** [review]
Looks good to me! Great work. 🥇

**Review by christianjokhaji — APPROVED** [review]

**Review by iDasher — APPROVED** [review]
Looks good, general test structure works for my code as well when not using slash commands.

**Review by sailoraria — APPROVED** [review]
looks good!

**Pull request merged**